datadog_lambda/tracing.py

```diff
@@ -1320,3 +1322,39 @@ def is_async(span: Span) -> bool:
                 e,
             )
             return False
+
+
+def emit_telemetry_on_exception_outside_of_handler(
+    context, exception, resource_name, handler_load_duration
+):
+    """
+    Emit an enhanced error metric and create a span for exceptions occuring outside of the handler
+    """
+    submit_errors_metric(context)
+
+    span = tracer.trace(
```

Comment:
We support an environment variable named `DD_TRACE_ENABLED`. I'm thinking we should probably consider that flag for this feature too? WDYT?